backend/llm/private_gpt4all.py

```diff
@@ -1,14 +1,13 @@
 from langchain.llms.base import LLM
 from langchain.llms.gpt4all import GPT4All
+from llm.openai import OpenAIBrainPicking
 from logger import get_logger
 from models.settings import LLMSettings
 
-from .base import BaseBrainPicking
-
 logger = get_logger(__name__)
 
 
-class PrivateGPT4AllBrainPicking(BaseBrainPicking):
+class PrivateGPT4AllBrainPicking(OpenAIBrainPicking):
```

Comment:
Shouldn't we actually keep `BaseBrainPicking` since GPT4All is not directly linked to OpenAI

Reply:
It uses all the same logic so feels silly to repeat. Maybe we should call the parent class something else.

Reply:
👍🏻 , even if just for the developer experience. At least if it's not really involving `OpenAI` I'd suggest to rename the class.